pyproject.toml

```diff
@@ -48,6 +48,7 @@ pre-commit = "^3.3.2"
 ipython = "^8.13.2"
 pytest-sugar = "^0.9.7"
 rich = "^13.3.5"
+docopt = "^0.6.2"
 
 [tool.poetry.scripts]
 cloudimagedirectory-transformer = "cloudimagedirectory.transformer:run"
```

Comment:
Does it make sense to register `cid2civ` here, the same way we did it with the transformer (and the old cli in the past)? :)